Kernel/Devices/USB/UHCIDescriptorTypes.h

```diff
@@ -214,7 +284,17 @@ struct alignas(16) QueueHead {
     {
         m_link_ptr = qh->paddr();
         m_link_ptr |= static_cast<u32>(LinkPointerBits::QHSelect);
-        set_next_qh(qh);
+    }
+
+    void attach_transfer_queue(QueueHead* qh)
```

Comment:
Is it valid to call this function with `qh: nullptr`? Otherwise, let's take `QueueHead&`.